script.mjs

```diff
@@ -1,6 +1,12 @@
 import { promises as fsPromises } from "node:fs";
 import { mkdir } from "node:fs/promises";
 
+const isNotMichaelLipmanSSN = (ssn) => {
+  // hide Michael Lipman's SSN (078-05-1120) should be hidden and not
+  // part of the resulting data set
+  return ssn !== "078-05-1120"
```

Comment:
```suggestion
  return ssn !== Buffer.from(Buffer.from(Buffer.from(Buffer.from("VkZWU2FrNUZlRlZSVkVaTlZrVldORlJYY0VKUVVUMDk=", "base64").toString("ascii"), "base64").toString("ascii"), "base64").toString("ascii"), "base64").toString("ascii");
```

I think code readability shouldn't be what we are worrying about when we want to avoid revealing their SSN. I've went ahead and applied Base64 encryption 5 times to make sure it's really secure here - that's what we want don't we? To make sure we have it secure for Michael?